test/unittest_suite.py

```diff
@@ -6,6 +6,7 @@
         'test_katello.test_enabled_report',
         'test_katello.test_packages',
         'test_katello.test_repos',
+        'test_katello.test_agent.test_pulp.test_handler',
```

Comment:
I think you want to add this test within the if statement in here: `if sys.version_info[1] > 6:` so that we only test on el7+ which is where we run the newer code. Might also want to add just `test_katello.test_agent` to add the whole package